modin/backends/pandas/parsers.py

```diff
@@ -300,7 +300,7 @@ def update_row_nums(match):
             **kwargs
         )
 
-        pandas_df = parser.read()
+        pandas_df = parser.read().dropna(how="all")
```

Comment:
What is this change for?

Reply:
It's a very obscure reason. I will document it there.

In excel if you create a row with only a border (no values), this parser will interpret that as a row of NaN values. Pandas discards these values.